pkgs/applications/terminal-emulators/blackbox-terminal/default.nix

```diff
@@ -57,7 +57,7 @@ stdenv.mkDerivation rec {
     pkg-config
     vala
     sassc
-    wrapGAppsHook
+    wrapGAppsHook4
     python3
     gtk3 # For gtk-update-icon-cache
```

Comment:
This should be fixed upstream so that gtk3 is not necessary ([example](https://gitlab.gnome.org/GNOME/swell-foop/-/merge_requests/34)). cc @chuangzhu